src/OrbitGl/TimeGraphLayout.h

```diff
@@ -16,6 +16,8 @@ class TimeGraphLayout {
   float GetTextCoresHeight() const { return core_height_ * scale_; }
   float GetThreadStateTrackHeight() const { return thread_state_track_height_ * scale_; }
   float GetEventTrackHeight() const { return event_track_height_ * scale_; }
+  float GetAllThreadsEventTrackHeight() const { return all_threads_event_track_height_ * scale_; }
```

Comment:
Nit: With this change, a future modification of event_track_height_ won't affect the AllThread's one, which could be strange because AllThread is also an event track. 

Feel free to disregard, but we might could change this function for GetAllThreadsExtraTrackHeight() and adding this height when the track is the AllThreads's one.

The cpp code could look like as:
```
float TimeGraphLayout::GetEventTrackHeightFromTid(int32_t tid) const {
  float track_height = GetEventTrackHeight();
  if (tid == orbit_base::kAllProcessThreadsTid) {
   track_height += GetAllThreadsExtraTrackHeight();
 } 
 return track_height;
}
```

Reply:
I agree with Pablo. Expressing the "AllThreads" height as a multiple of the normal track height could also work, in this case we'd have a multiplier of 2. This way the relative heights is maintained if we change the normal track height.

Reply:
Done. I changed it to be a scale as Pierric suggested.

Reply:
I thought about this other possibility, seems also good for me. :)